launch.sh

```diff
@@ -10,11 +10,56 @@ until pg_isready; do
 done
 
 echo "Postgres is up, loading chado"
-su postgres -c 'createdb apollo'
-su postgres -c 'createdb chado'
+
+if [[ "${WEBAPOLLO_DB_HOST}" == "" ]]; then
+	WEBAPOLLO_DB_HOST=127.0.0.1
```

Comment:
there are easier ways to do this https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts/122848#122848